src/components/JdsDataTable/DataTable.vue

```diff
@@ -194,15 +212,20 @@ export default {
       default: 'id'
     }, 
 
-     /** NOTE: 
-      * this property will be changed 
-      * and/or adjusted during implementation and 
-      * integration with JdsPagination component
-     */
+   /**
+    * Pagination property
+    * <br><br>
+    * for more information, check
+    * `JdsPagination` component documentation
+    */
     pagination: {
       type: Object,
       default: () => ({
+        currentPage: 1,
+        totalRows: 0,
         itemsPerPage: 10,
+        itemsPerPageOptions: [10, 20, 30, 40, 50],
+        disabled: false
       })
     }
```

Comment:
Expecting a default value for prop that is a typeof `Object` has it perks.
For example, if i use this component like this.
```vue
<template>
  <jds-data-table
    :pagination="{
      itemsPerPage: 5
    }"
  />
</template>
```
That would remove the remaining default value, IIRC?. As in, `currentPage`, `totalRows`, etc wont exists within the `pagination` object.

This shouldn't be a concern if:
- each key in `pagination` is noted as mandatory to be defined.
- `JdsPagination` has the same default with what defined above.

Reply:
thank you for the insight, fixed [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/100/commits/f5d5f72c23fb5d9a4b4501e8a066bcb4c7e78c4c) @adrianpdm 👍🏼